src/cmd/build.js

```diff
@@ -1,30 +1,31 @@
 /* @flow */
 import path from 'path';
-import minimatch from 'minimatch';
 import {createWriteStream} from 'fs';
-import streamToPromise from 'stream-to-promise';
+
+import minimatch from 'minimatch';
 import {fs} from 'mz';
+import streamToPromise from 'stream-to-promise';
 import parseJSON from 'parse-json';
 
 import defaultSourceWatcher from '../watcher';
 import {zipDir} from '../util/zip-dir';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import {prepareArtifactsDir} from '../util/artifacts';
 import {createLogger} from '../util/logger';
-import {UsageError} from '../errors';
+import {UsageError} from '../errors'; //eslint-disable-line import/order
```

Comment:
you will run into this with all flow type imports. We kept those separate to make it obvious that they are only for flow but I think it's fine to group them with the other parent/sibling imports.